### ISSUE TITLE
Stable version for PHP

### ISSUE THREAD

**Issue by antonioribeiro** [snowplow/referer-parser#95]
Are you still developing the PHP version? Asking because last (numeric) stable version is 326 commits old and has no PHP on it. Tagged version java-scala-0.2.3 is up to date with master, but, looking at the name of the tag, it doesn't look like you care about PHP anymore.

Cheers!

**Comment by alexanderdean** [comment]
I've added a tag now! The PHP one is up-to-date with the latest database of referers.

**Comment by antonioribeiro** [comment]
Thank you!

**Comment by alexanderdean** [comment]
No problem!

**Comment by antonioribeiro** [comment]
Turns out that you still have no versions published in Packagist: 

https://packagist.org/packages/snowplow/referer-parser. 

And, even if you tell Packagist to refresh it, I'm not sure it will undestand the version `php-0.1.0` as stable, according to https://getcomposer.org/doc/02-libraries.md#tags.

To test it I tagged (and forced an update to) another package of mine and the tag is not even showing in the list: https://packagist.org/packages/pragmarx/datatables.
